test/WebSites/ActionResultsWebSite/Controllers/ActionResultsVerificationController.cs

```diff
@@ -108,5 +119,20 @@ private DummyClass CreateDummy()
                 SampleString = "Foo"
             };
         }
+
+        private DisposableDummy CreateDisposableDummy()
+        {
+            return new DisposableDummy();
+        }
+
+        private class DisposableDummy : IDisposable
+        {
+            public static int disposeCallCount = 0;
```

Comment:
Yeah I'm sure Ryan has some ideas :smile: The problem with statics is that they their state is maintained until the entire process exits, and the process presumably has the lifetime of the entire set of tests. So, if one test were to "dirty up" some state, it could affect other tests. In this particular case, as the code is right now, it's probably not an issue. But if later another test is added that uses this same type for convenience, weird things will happen.